test/test_torch.py

```diff
@@ -1819,6 +1819,25 @@ def _test_bitwise_not(self, device):
     def test_bitwise_not(self):
         self._test_bitwise_not(self, 'cpu')
 
+    @staticmethod
+    def _test_logical_not(self, device):
+        for dtype in (torch.bool, torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64,
```

Comment:
can you use https://github.com/pytorch/pytorch/blob/9588cd921efd051a982188220b8215914bd5790e/torch/testing/__init__.py#L94-L102?